commit e4a103b63811c7042f89e8d608cf47d463cdad57 (refs/changes/76/2776/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-10-11T18:30:06-07:00 (2 years ago)
    
    ZAPI-807 allow sdc-vmapi's rsync-to to work with a VMAPI instance *not* named "vmapi0"
    Reviewed by: Julien Gilli <julien.gilli@joyent.com>
    Approved by: Julien Gilli <julien.gilli@joyent.com>

diff --git a/tools/rsync-to b/tools/rsync-to
index a510662..4e5818f 100755
--- a/tools/rsync-to
+++ b/tools/rsync-to
@@ -13,14 +13,19 @@
 # Rsync the master in this working copy to the install on the given HN.
 #
 
-#set -o xtrace
+if [[ -n "$TRACE" ]]; then
+    export PS4='[\D{%FT%TZ}] ${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
+    set -o xtrace
+fi
 set -o errexit
+set -o pipefail
+
 
 TOP=$(cd $(dirname $0)/../; pwd)
 NODE="root@$1"
 
 if [[ -z "$VMAPI_ZONE" ]]; then
-    VMAPI_ZONE=$(ssh $NODE "vmadm lookup -1 alias=vmapi0" 2>/dev/null)
+    VMAPI_ZONE=$(ssh $NODE "vmadm lookup -1 alias=~^vmapi" 2>/dev/null)
 fi
 echo "VMAPI_ZONE: $VMAPI_ZONE"
 
